server/websocket.go

```diff
@@ -744,7 +750,10 @@ func (p *Plugin) handleJoin(userID, connID, authSessionID string, joinData Calls
 			})
 		}
 
-		p.LogDebug("session has joined call", "userID", userID, "sessionID", connID, "channelID", channelID, "callID", state.Call.ID)
+		p.LogDebug("session has joined call",
```

Comment:
I understand that this is logged only in the log level in the server is set to Debug, right?

Reply:
Yep